interpreter.py

```diff
@@ -198,6 +236,10 @@ def exec_operation(self, instruction):
             var_val = self.lookup_var(args[0])
             assert isinstance(var_val, Float)
             return Float(var_val.value)
+        elif opcode == LLVMZExt:
+            var_val = self.lookup_var(args[0])
+            assert isinstance(var_val, Integer)
+            return Integer(var_val.value)
```

Comment:
I'll have to think about that, but in this case, I need to modify some values. For instance, I have alloca and store instructions: after allocating memory for a variable, another instruction (store) requires me to store a new value for it.